internal/command/volumes/create.go

```diff
@@ -153,3 +122,42 @@ func runCreate(ctx context.Context) error {
 
 	return printVolume(out, volume)
 }
+
+func confirmVolumeCreate(ctx context.Context, appName string) (bool, error) {
+	var volumeName = flag.FirstArg(ctx)
+
+	// If the Yes flag has been supplied we skip the warning entirely, so no
+	// need to query for the set of volumes
+	if !flag.GetYes(ctx) {
```

Comment:
Avoid the nested ifs in case like this, see [reduce nesting, reduce complexity](https://testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html), this is the same principle used for [shortcircuit'ing error flow](https://google.github.io/styleguide/go/decisions#indent-error-flow)

Reply:
Thank you! Done [=